README.md

```diff
@@ -1,23 +1,23 @@
 # What is OceanBase database
 
-OceanBase Database is a native distributed relational database. It is developed entirely by Alibaba and Ant Group. OceanBase Database is built on a common server cluster. Based on the Paxos protocol and its distributed structure, OceanBase Database provides high availability and linear scalability. OceanBase Database is not dependent on specific hardware architectures.
+OceanBase Database is a native distributed relational database. It is developed entirely by Alibaba and Ant Group. OceanBase Database is built on a common server cluster. Based on the Paxos protocol and its distributed structure, OceanBase Database provides high availability and linear scalability. OceanBase Database is not dependent on any specific hardware architecture.
 
 OceanBase Database has these features:
 
 - High availability
-    Single server failure recovers automatically. OceanBase Database supports cross-city disaster tolerance for multiple IDCs and zero data loss. OceanBase Database meets the financial industry Level 6 disaster recovery standard (RPO=0, RTO<=30 seconds).
+    OceanBase Database recovers from single server failure automatically. OceanBase Database supports cross-city disaster tolerance for multiple IDCs and zero data loss. OceanBase Database meets the financial industry Level 6 disaster recovery standard (RPO=0, RTO<=30 seconds).
 - Linear scalability
-    OceanBase Database scales transparently to applications and balances the system load automatically. Its cluster can contain more than 1500 nodes. The data volume can reach petabytes. The records in a single table can be more than a trillion rows.
-- Highly compatible with MySQL
-    OceanBase Database is compatible with MySQL protocol and syntax. You can access to OceanBase Database by using MySQL client.
+    OceanBase Database scales transparently to applications and balances the system load automatically. It can manage more than 1500 nodes in a cluster. The data volume can reach petabytes. The records in a single table can be more than a trillion rows.
+- High compatibility with MySQL
+    OceanBase Database is compatible with MySQL protocol and syntax. You can access OceanBase Database by MySQL client.
 - High performance
     OceanBase Database supports quasi memory level data change and exclusive encoding compression. Together with the linear scalability, OceanBase Database provides high performance.
 - Low cost
-    OceanBase Database uses PC servers and cheap SSDs. Its high storage compression ratio and high performance also reduce the storage cost and the computing cost.
+    OceanBase Database can leverage PC servers and cheap SSDs. Its high storage compression ratio and high performance also reduce the storage and computing costs.
```

Comment:
Thanks for your contribution. How about
OceanBase Database leverages PC servers and cheap SSDs.